### PR TITLE
Browser matrix

### PR DESCRIPTION
**Pull request by AbdulrhmnGhanem**
Run integration tests on Chrome, Firefox, and Edge.

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -232,12 +247,14 @@ jobs:
...
-          command: yarn e2e --record --browser chrome
-          working-directory: frontend
           # wait for 2 minutes for the server to respond
           wait-on: 'http://kitspace.test:3000'
           # This value is in seconds
-          wait-on-timeout: 120
+          wait-on-timeout: 120         
+          working-directory: frontend
+          record: true
+          browser: ${{ matrix.browsers }}
```
Comment (kasbah) [inline .github/workflows/docker.yml:256]: Why did you change this to use `command` before? I was non-command before then you changed it to be command and now back again. Just trying to understand...

Comment (AbdulrhmnGhanem) [reply .github/workflows/docker.yml:256]: If the `command` key is present, the run options e.g., `record`, and `browser` are ignored.
In 6356f81088aad69abb363e3adf685d4f700fb1c4, I thought there was a problem with the run options, turned out I used `matrix.browser` instead of `matrix.browsers`.

**Comment by AbdulrhmnGhanem** [comment]
You can squash this, just split it into multiple commits to see if any specific step will cause errors.

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -180,6 +180,10 @@ jobs:
     runs-on: ubuntu-20.04
     needs:
       - build
+    strategy:
+      fail-fast: false
+      matrix:
+        browsers: [chrome, firefox, edge]
```
Comment (kasbah) [inline .github/workflows/docker.yml:186]: I think we should run it in electron too. Since in practice that surfaced some potential bugs already.

**Comment by kasbah** [comment]
While you are looking at tests. Can we reduce the flakiness test to a warning for now until we can completely fix the flakiness?

**Comment by AbdulrhmnGhanem** [comment]
> While you are looking at tests. Can we reduce the flakiness test to a warning for now until we can completely fix the flakiness?

I don't have managing the dashboard, you can configure it as shown [here](https://docs.cypress.io/guides/dashboard/flaky-test-management#GitHub).

File: .github/workflows/docker.yml (hunk 2 above)
Comment (AbdulrhmnGhanem) [reply .github/workflows/docker.yml]: Resolved via 6c2ffac.

**Comment by kasbah** [comment]
Thanks, I've disabled it. I disabled the pull-request comments as well.
